.gitignore

```diff
@@ -15,8 +15,8 @@ cscope.out
 /external
 /.context
 /.config
+/.depend
```

Comment:
dup to line 1

Reply:
oops.
it's one of reasons one should not make changes without mentioning it in the commit message. :-)